src/client/app.js

```diff
@@ -541,7 +542,8 @@ class CellLabelingApp {
                             label: x.label
                         }
                     }),
-            roi_extra
+            roi_extra,
+            duration: (Date.now() - this.labelingStart) / 1000
```

Comment:
So the time would be, from the moment they make their account till they finish or as long as they have it open in their browser? As in if you open a browsers window that is `labelingStart` and closing it is `duration` or is it "I labeled all the regions available to me? Or, as I think it is below, is it the time it took to label one region?

Reply:
It is the time it took to label 1 region, from the time the region first starts to load to submission

If the region loads, then they walk away, then submit, it would be this total time.

It's got a few problems, maybe an improvement would be to start the clock when they click the projection for the first time on a new region.

Reply:
That could work and may be more accurate. However, I'm not sure it would be that much of a difference between "first click" and "load region". I'm fine with "load region" to give us an estimate.